mods/cnc/maps/nod02a/nod02a.lua

```diff
@@ -234,7 +234,7 @@ Tick = function()
 		Nod.MarkCompletedObjective(NodObjective1)
 	end
 
-	if DateTime.GameTime % DateTime.Seconds(3) == 0 and Barracks.IsInWorld then
+	if DateTime.GameTime % DateTime.Seconds(3) == 0 and Barracks.IsInWorld and Barracks.Owner == gdi then
```

Comment:
I wonder which is more efficent
´´´
if DateTime.GameTime % DateTime.Seconds(3) == 0 and Barracks.IsInWorld and Barracks.Owner == gdi
´´´
or
´´´
if Barracks.IsInWorld and Barracks.Owner == gdi and DateTime.GameTime % DateTime.Seconds(3) == 0
´´´

and yes it doesnt really matter since the performance gain either way would be miniscule, but still makes me wonder.